addons/website/views/snippets/snippets.xml

```diff
@@ -301,11 +301,11 @@
 
     <div data-js="sizing_y"
         data-selector="section, .row > div, .parallax, .s_hr, .s_btn, .carousel-item"
-        data-exclude="section:has(> .carousel), .o_gallery .carousel-item"/>
+        data-exclude="section:has(> .carousel), .o_gallery .carousel-item, .s_company_team_member > div"/>
```

Comment:
I think it should make sense to excluse s_col_no_resize instead, like for sizing_x. You can then remove the new s_company_team_member class.